### DISCUSSION TITLE
Disable flash

### DISCUSSION THREAD

**Discussion by Tobbe** [wezterm/wezterm#1046] [Q&A] [answered]
Hi, and first of all, thanks a lot for creating wezterm.

Is it possible to turn off the flash happens for example when you try to tab-complete a path and there are multiple matches? At first I was going to ask "Is it possible to disable the visual bell", but now I'm not sure that's the correct terminology as https://github.com/wez/wezterm/issues/3 makes it sound as there is no visual bell implemented. But I hope you understand what I'm asking.

**Comment by wez** [comment]
wezterm doesn't have any flashing or bell functionality.  Perhaps it is the shell or application that you are using that is doing that?

**Reply by Tobbe** [reply]
Sorry, I should have provided more context.

I'm on Windows 10. Using git-bash as my shell. I found the easiest way to trigger the flash is to just press backspace at the command prompt.

Here's my config file

```lua
local wezterm = require "wezterm"
local gitbash = {"C:\\Program Files\\Git\\bin\\bash.exe", "-i", "-l"}
return {
    color_scheme = "Dracula",
    default_prog = gitbash,
    window_background_opacity = 0.9,

    font_size = 9.0,

    launch_menu = {
        {
            label = "Powershell",
            args = {"powershell"},
        },
        {
            label = "Git Bash",
            args = gitbash,
        },
    },
}
```

I don't get that flash when running the same shell in another terminal (neither in ConEmu nor MinTTY)

**Reply by Tobbe** [reply]
Here's a screen recording when I press backspace

https://user-images.githubusercontent.com/30793/129942043-54bde9e1-9d8c-4a1a-a460-45484618ba1e.mp4

**Reply by wez** [reply]
I think you need to configure your bash/inputrc to control this: there is no code in wezterm that does this; I hate visual bells!

**Comment by kar1** [comment, marked as answer]
FWIW https://github.com/microsoft/terminal/issues/7200

> To fix: nano ~/.inputrc and add the text set bell-style none or set bell-style audible. Press Ctrl + O and Ctrl + X to Save and Exit.
> 
> I found this behaviour to be really distracting/disurbing.
> The regular Git Bash terminal does exhibit this behaviour, though I can confirm that /c/Program Files/Git/etc/.inputrc does contain set bell-style visible (and that command is not overwritten by anything in .inputrc.
> 
> Note that set bell-style audible does not give a ping sound in Git Bash in Windows Terminal, but it does do so in regular Git Bash.
